fre/make/create_checkout_script.py

```diff
@@ -12,8 +12,30 @@
 fre_logger = logging.getLogger(__name__)
 
 def checkout_create(yamlfile, platform, target, no_parallel_checkout, jobs, execute, verbose):
```

Comment:
Can some of these arguments made optional? For example, 
no_parallel_checkout: bool = False?

(Also, is type hinting?)

Reply:
These functions are used by fremake right?  Maybe that should be added in here somewhere.